src/overmap.cpp

```diff
@@ -1817,7 +1816,7 @@ bool overmap::generate_sub(int const z)
                 ter( i, j, z ) = oter_id( "spiral_hub" );
                 add_mon_group( mongroup( mongroup_id( "GROUP_SPIRAL" ), i * 2, j * 2, z, 2, 200 ) );
             } else if ( oter_above == "silo" ) {
-                if (rng(2, 7) < abs(z) || rng(2, 7) < abs(z)) {
+                if( rng( 2, 7 ) < abs( z ) ) {
```

Comment:
Technically this is not the same. Each call to `rng` can return a different number, so it's possible that the first part of the condition is `false`, while the second part is `true`. Now the chance of branching is halved.

Reply:
Yes, but seeing identical subexpressions is strange - could it be a typo in initial implementation?

Reply:
It is strange, but I suspect the original author found this to be the best way to express what they were trying to do.

Reply:
I will rollback these changes then.